ethcore/src/engines/authority_round/mod.rs

```diff
@@ -47,6 +47,9 @@ use types::header::{Header, ExtendedHeader};
 use types::ancestry_action::AncestryAction;
 use unexpected::{Mismatch, OutOfBounds};
 
+#[cfg(not(feature = "time_checked_add"))]
```

Comment:
Is this referring to rust feature or crate feature? I think for rust feature, we may need to define it to be used here:

```
#![cfg_attr(feature = "time_checked_add", feature(time_checked_add)]
```

Reply:
Yeah, it is a rust feature.

It is a good idea I will change it will definitely be much cleaner :+1: